### PR TITLE
Update Yammer in M365 group auto renewal

### PR DESCRIPTION
**Pull request by slowik84**
added Yammer auto renewal feature based on https://docs.microsoft.com/en-us/azure/active-directory/enterprise-users/groups-lifecycle and removed note about file add

### CODE REVIEW

**Comment by MikePlumleyMSFT** [comment]
@ArunMSFT can you review?

File: microsoft-365/solutions/microsoft-365-groups-expiration-policy.md [hunk 1]
```diff
@@ -33,8 +33,7 @@ Groups that are actively in use are renewed automatically. Any of the following
 - SharePoint - view, edit, download, move, share, or upload files. (Viewing a SharePoint page does not count as an action for automatic renewal.)
 - Outlook - join group, read or write group message from the group, and like a message (Outlook on the web).
 - Teams - visiting a teams channel.
-
-Note that the only Yammer activity that will trigger an automatic group renewal is the upload of a document to SharePoint within the community.
+- Yammer: View a post within a Yammer community or an interactive email in Outlook
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/microsoft-365-groups-expiration-policy.md]:
```suggestion
- Yammer - View a post within a Yammer community or an interactive email in Outlook.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review @slowik84, kindly commit the minor suggested change. Many thanks.

**Comment by yogkumgit** [comment]
@slowik84 Gentle reminder, Please implement the suggested changes. Thanks

**Comment by ArunMSFT** [comment]
Looks good to me.

@Mike ***@***.***> FYI

**Comment by ArunMSFT** [comment]
Thanks so much, Mike Plumley, for making this change. One request...SharePoint, Outlook, and Teams are all followed by a hyphen before the statement which explains their renewal activities. Can we do the same for Yammer in this particular article? Currently I see a colon in GitHub, presumably because it was copy&pasted from the other article. Thanks!

Mike McLean

From: Arunkumaran Varadharajan ***@***.***>
Sent: Wednesday, March 23, 2022 7:50 AM
To: MicrosoftDocs/microsoft-365-docs ***@***.***>; MicrosoftDocs/microsoft-365-docs ***@***.***>; Mike McLean ***@***.***>
Cc: Mention ***@***.***>
Subject: RE: [MicrosoftDocs/microsoft-365-docs] Update Yammer in M365 group auto renewal (PR #7982)

Looks good to me.

@Mike ***@***.***> FYI

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Suggestion updated in accordance with @ArunMSFT's observation. @slowik84, kindly commit the suggested change.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: microsoft-365/solutions/microsoft-365-groups-expiration-policy.md [hunk 2]
```diff
@@ -33,8 +33,7 @@ Groups that are actively in use are renewed automatically. Any of the following
 - SharePoint - view, edit, download, move, share, or upload files. (Viewing a SharePoint page does not count as an action for automatic renewal.)
 - Outlook - join group, read or write group message from the group, and like a message (Outlook on the web).
 - Teams - visiting a teams channel.
-
-Note that the only Yammer activity that will trigger an automatic group renewal is the upload of a document to SharePoint within the community.
+- Yammer - View a post within a Yammer community or an interactive email in Outlook.
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/microsoft-365-groups-expiration-policy.md:36]:
```suggestion
- Yammer - view a post within a Yammer community or an interactive email in Outlook.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@slowik84, apologies - one more minor change for consistency, please.

**Comment by yogkumgit** [comment]
@slowik84 Gentle reminder. Thanks

**Comment by yogkumgit** [comment]
@slowik84 closing the PR, as suggested changes are not implemented. Please feel free to reopen it, in case you wish to take an action. Thanks

**Pull request closed without merging**